app/assets/javascripts/discourse/app/components/bread-crumbs.hbs

```diff
@@ -17,17 +17,7 @@
   {{#if breadcrumb.hasOptions}}
     <li
       style={{html-safe
-        (if
-          breadcrumb.category
-          (concat
-            "--category-bg-color:#"
-            breadcrumb.category.color
-            "; "
-            "--category-text-color:#"
-            breadcrumb.category.text_color
-            ";"
-          )
-        )
+        (if breadcrumb.category (category-variables breadcrumb.category))
```

Comment:
Just want to confirm that this is a refactoring change and not really related to the fix here?

Reply:
Refactoring and a bug fix. The category colors were not rendered correctly, because we are now injecting stylesheets to color the badges in an initializer, but also because the variables names are not consistent.

Reply:
Created https://github.com/discourse/discourse/pull/24527